File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        ans = 0
        for i in range(len(nums)):
            if nums[i] == 1:
                count += 1
            else:
                count = 0
            ans = max(ans, count)
        return ans
                
                
        

File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
       int max_count = 0;
        int maxi = INT_MIN;
        for(int i=0;i<nums.size();i++){
            if(nums[i]==1) max_count++;
            else {
                max_count = 0;
            }
            maxi = max(maxi, max_count);
        }
        return maxi;
    }
};